content/blog/broom-0-7-0/index.Rmd

```diff
@@ -158,12 +158,15 @@ tidyverse domain to [broom.tidymodels.org](https://broom.tidymodels.org/).
 
 ### Looking Forward
 
-Most notably, **the broom dev team generally will no longer be adding new tidier 
+Most notably, **the broom dev team generally is changing the process to add new tidying 
 methods to the package.** Instead, we ask that issues/PRs requesting support for 
 new model objects be directed to the model-owning package (i.e. the package 
-that the model is exported from) rather than to broom. 
+that the model is exported from) rather than to broom. If the maintainers of 
+those packages are unable or unwilling to provide tidying methods in the
+model-owning package, we would then welcome a PR with a new tidier to broom.
```

Comment:
This reads as way too permissive to me. 

```suggestion
model-owning package, it might be possible to add the new tidier to broom. broom is near its limit of tidiers; adding more may make the package unsustainable. 
```